hc/settings.py

```diff
@@ -131,10 +113,20 @@
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
     'compressor.finders.CompressorFinder',
 )
+STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
+
 COMPRESS_OFFLINE = True
 
 EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
 
+# Update database configuration with $DATABASE_URL.
+# if os.environ.get('HEROKU'):
+#     db_from_env = dj_database_url.config()
+#     DATABASES['default'].update(db_from_env)
+
+db_from_env = dj_database_url.config()
+DATABASES['default'].update(db_from_env)
```

Comment:
We could also try 
if 'HEROKU' in os.environ:
      db_from_env = dj_database_url.config()
      DATABASES['default'].update(db_from_env)